numpy/core/src/umath/loops.c.src

```diff
@@ -786,6 +795,14 @@ NPY_NO_EXPORT void
     }
 }
 
+NPY_NO_EXPORT void
+@TYPE@_isfinite(char **args, npy_intp *dimensions, npy_intp *steps, void *NPY_UNUSED(func))
+{
+    OUTPUT_LOOP {
+        *((@type@ *)op1) = 1;
```

Comment:
Having a `np.isna` function that handles both nans and NATs might be good. I'm thinking of the future when we might have "missing value" dtypes. Also note that all the `nan*` functions can also be easily modified to handle NA, or even finite values. Makes me wonder if those functions are too specific.